eng/Subsets.props

```diff
@@ -261,7 +253,17 @@
     <PkgprojProjectToBuild Include="$(InstallerProjectRoot)pkg\projects\**\*.pkgproj" SignPhase="MsiFiles" BuildInParallel="false" />
     <ProjectToBuild Include="@(PkgprojProjectToBuild)" Pack="true" Category="host" />
   </ItemGroup>
-  
+
+  <!-- Libs test and packaging -->
+  <ItemGroup Condition="$(_subset.Contains('+libs.tests+'))">
+    <ProjectToBuild Include="$(LibrariesProjectRoot)tests.proj" Category="libs" Test="true" />
+  </ItemGroup>
+
+  <ItemGroup Condition="$(_subset.Contains('+libs.packages+'))">
```

Comment:
Libs packages should be before libs.tests, after libs.pretest.

Reply:
Good